### DISCUSSION TITLE
WASM doesn't load properly and weird output in terminal

### DISCUSSION THREAD

**Discussion by SivertBrekke** [unoplatform/uno#11549] [Q&A]
Hello!

I am working on an app utilizing UNO platform and for some reason when I try debugging the WASM app I get this output;
`fail: Microsoft.WebAssembly.Diagnostics.DevToolsProxy[0] sending error response for id: msg-39F396592F01893B845B70472A6E63F1:::1017 -> [Result: IsOk: False, IsErr: True, Value: , Error: {   "code": -32601,   "message": "'DotnetDebugger.setDebuggerProperty' wasn't found" } ]
`
I've scoured the internet for a solution for this but I am stuck, does anyone have any pointers/tips to what's wrong? :D

**Comment by Rytisgit** [comment]
can you create and share a minimal reproduction of the issue?

**Reply by jeromelaban** [reply]
Can you also specify which version of visual studio and browser you are using?

**Reply by SivertBrekke** [reply]
I am using Visual Studio 2022 and Chrome 110.0.5481.178. I will provide minimal reproduction as soon as I'm able!

**Reply by jeromelaban** [reply]
Thanks. Which version of Visual Studio 2022 ?
